superflore/generators/ebuild/ebuild.py

```diff
@@ -157,13 +157,11 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
         ret += " license\n\n"
 
         # EAPI=<eapi>
-        ret += "EAPI=" + self.eapi + "\n\n"
-        """
-        @todo: don't hard code this
-        """
+        ret += "EAPI=" + self.eapi + "\n"
+        ret += "PYTHON_COMPAT=( python{2_7,3_5} )\n\n"
```

Comment:
As it is now, there is no such list available (and, due to the sheer number of packages, such a list is likely to be taken on a case-by-case basis). So, I will remove the python3 flag only for those packages which are known to fail. Although Python 3 does not *work* with all packages, it will still install all packages (as exhibited by my ability to install ros-lunar with 3.5).

We'll simply add a disclaimer that Python 3 is not guaranteed to work for all packages.